lib/config-generator.js

```diff
@@ -558,7 +564,7 @@ class ConfigGenerator {
     buildCacheConfig() {
         const cache = {};
 
-        cache.type = 'filesystem';
+        cache.type = /** @type {const} */ ('filesystem');
```

Comment:
this syntax with `/** @type */` followed by a parenthesized expression as the JSDoc syntax for a cast.
Using `const` as type allows to do the same than [`as const`](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-inference) in typescript